qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -34,6 +41,22 @@ public void setFileUrl(String fileUrl) {
 		WebElement element = driver.findElement(By.xpath("//input[@type='file']"));
 		element.sendKeys(fileUrl);
 	}
+	
+	private String createTemporaryFile() throws IOException {
```

Comment:
Generally, I try to keep `private` functions near the bottom. Yes, this should be `private` it's here to implement part of a flow... no one else should call `new AttachmentsPage().createTemporaryFile()`

Reply:
@ibacher , I have attended to your reviews and pushed a commit. Could you get some time and review it please? Thank you very much.